README.md

```diff
@@ -1,16 +1,23 @@
-# litentry-parachain
+# Litentry Parachain
```

Comment:
Can you add some cool Logos of litentry parachain? 😎  Can be found here https://brandpad.io/litentrytechnologies/#